src/stage1/parse_f128.c

```diff
@@ -47,7 +56,6 @@
 
 #define DECIMAL_DIG 36
 
-
 #if __BYTE_ORDER == __LITTLE_ENDIAN
```

Comment:
This check is better written as follows, coding defensively is a must when you throw the preprocessor in the mix.
```c
#if defined(__BYTE_ORDER) && __BYTE_ORDER == __LITTLE_ENDIAN
```

Reply:
Added extra checks here too, thanks.